iontestdata/bad/README.md

```diff
@@ -65,18 +65,18 @@ data are available.
 
 decimalExpTooLarge.10n
 ----------------------
-This file contains a decimal who's exponent exceeds the length defined by the
+This file contains a Decimal who's exponent exceeds the length defined by the
```

Comment:
While you're here, would you mind correcting "who's" -> "whose"?

Reply:
Done